image_proc/src/image_proc.cpp

```diff
@@ -75,33 +54,21 @@ int main(int argc, char * argv[])
 
   rclcpp::init(argc, argv);
 
-  std::string camera_namespace;
-  if (rcutils_cli_option_exist(argv, argv + argc, "--camera_namespace")) {
-    camera_namespace = std::string(rcutils_cli_get_option(argv, argv + argc, "--camera_namespace"));
-  }
-  else {
-    print_usage();
-    return 0;
-  }
+  std::string camera_namespace = "image";
 
   rclcpp::executors::SingleThreadedExecutor exec;
   const rclcpp::NodeOptions options;
 
   // Debayer nodelet, image_raw -> image_mono, image_color
   auto debayer_node = std::make_shared<image_proc::DebayerNode>(options);
-  debayer_node->declare_parameter("camera_namespace", camera_namespace);
-  
 
   // Rectify nodelet, image_mono -> image_rect
   auto rectify_mono_node = std::make_shared<image_proc::RectifyNode>(options);
-  rectify_mono_node->declare_parameter("camera_namespace", camera_namespace);
-  rectify_mono_node->declare_parameter("image_mono", "/image_mono");
-  
+  rectify_mono_node->set_parameter(rclcpp::Parameter("image_topic", "/image_mono"));
```

Comment:
Please revert this parameter ASAP, we should not be casually changing parameter names, this could make a huge issue across a lot of peoples' software after release

Reply:
This was my fault. Should have caught that.

Reply:
Ok will try to keep them.